### DISCUSSION TITLE
How to publish views?

### DISCUSSION THREAD

**Discussion by jordivela007** [omnia-digital/livewire-calendar#14] [Q&A] [answered]
I need to change the day.blade.php because I do not use id index. I use UUID

**Comment by joshtorres** [comment, marked as answer]
You should be able to just do `php artisan vendor:publish` and then search for "calendar". Then, select the "Omnia\LivewireCalendar\LivewireCalendarServiceProvider". That will publish all of the views to "resources/views/vendor/livewire-calendar" and you can edit them however you'd like from there.
![image](https://github.com/omnia-digital/livewire-calendar/assets/5092957/32f00f9e-dd5f-491a-b74b-b346ea5643c7)
